exercises/tournament/tournament_test.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'tournament'
+
+# Test data version:
+# 0a51cfc
+class TournamentTest < Minitest::Test
+
+  def test_typical_input
+    # skip
+    input = <<-INPUT
+Allegoric Alaskans;Blithering Badgers;win
+Devastating Donkeys;Courageous Californians;draw
+Devastating Donkeys;Allegoric Alaskans;win
+Courageous Californians;Blithering Badgers;loss
+Blithering Badgers;Devastating Donkeys;loss
+Allegoric Alaskans;Courageous Californians;win
+INPUT
+    actual = Tournament.tally(input)
+    expected = <<-TALLY
+Team                           | MP |  W |  D |  L |  P
+Devastating Donkeys            |  3 |  2 |  1 |  0 |  7
+Allegoric Alaskans             |  3 |  2 |  0 |  1 |  6
+Blithering Badgers             |  3 |  1 |  0 |  2 |  3
+Courageous Californians        |  3 |  0 |  1 |  2 |  1
+TALLY
+    assert_equal expected, actual
```

Comment:
It will only work if there is no internal indentation happening.  True in this case, I think.